src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,25 @@
+package seedu.addressbook.data.person;
+
+public class Contact {
```

Comment:
I think maybe we can make Contact an abstract class so the method isValid() can be implemented in Contact without returning any value.